[build_grimoirelab] --build is no longer a default action

jgbarah opened the pull request:
When installing from already produced packages, --build should no longer be a default action. Therefore, now if you want --build and --install, you need to specify both.

acs commented on utils/build_grimoirelab:
```diff
@@ -80,7 +80,6 @@ def parse_args ():
     args_actions = parser.add_argument_group("Actions", "Actions to perform")
     args_actions.add_argument("--build", action='store_true',
-                            default=True,
                             help="Build modules. It is the default action.")
```
"Build modules. It is the default action."

The last "It is ..." must be removed, no?

jgbarah commented:
I hope this new commit addresses your concerns. In fact, `--build` should be the default action, but only if no other action was defined (so that for example you can `--install` already produced pkgs)
